terraform/fec_rds.tf

```diff
@@ -73,10 +70,10 @@ resource "aws_db_instance" "rds_production" {
   }
   engine = "postgres"
   engine_version = "9.6.1"
-  instance_class = "db.t2.micro"
-  allocated_storage = 10
-  name = "fec_production"
-  username = "${var.rds_production_username}"
+  instance_class = "db.r3.2xlarge"
+  allocated_storage = 1650
```

Comment:
I just spoke with Rohan and Wei at FEC, we'd like to bump this up to 2 TB (`2000`) for all environments.